Content.Server/GameTicking/Rules/RevolutionaryRuleSystem.cs

```diff
@@ -86,42 +84,28 @@ protected override void ActiveTick(EntityUid uid, RevolutionaryRuleComponent com
 
     private void OnRoundEndText(RoundEndTextAppendEvent ev)
     {
-        var revsLost = CheckRevsLose();
-        var commandLost = CheckCommandLose();
         var query = AllEntityQuery<RevolutionaryRuleComponent>();
         while (query.MoveNext(out var headrev))
         {
-            if (commandLost && !revsLost)
-            {
-                ev.AddLine(Loc.GetString("rev-won"));
-            }
-            else if (!commandLost && revsLost)
-            {
-                ev.AddLine(Loc.GetString("rev-lost"));
-            }
-            else if (commandLost && revsLost)
-            {
-                ev.AddLine(Loc.GetString("rev-stalemate"));
-            }
-            else if (!commandLost && !revsLost)
-            {
-                ev.AddLine(Loc.GetString("rev-reverse-stalemate"));
-            }
+            var revsLost = CheckRevsLose();
```

Comment:
since CheckRevsLose and CheckCommandLose dont depend on the rule (which CheckRevsLose seems it should...) just call them before the loop like before

Reply:
Nevermind it doesn't, I have lost my mind